src/components/AdminPage.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import classes from "./AdminPage.module.css";
+import { BsListOl } from "react-icons/bs";
+import { FiEyeOff, FiEye } from "react-icons/fi";
+
+const AdminPage = () => {
+  return (
+    <div className={classes.admin_container}>
```

Comment:
Reminder for this page, as well for the other, wrap those pages into some `Layout` component that will have all the repetetive elements (Navbar, background, padding etc.)